app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self,
+                 name: str):
```

Comment:
I would left args in one line

Reply:
matter of personal code style. I prefer writing constructors in this manner.